src/config.ts

```diff
@@ -41,7 +41,10 @@ export const EGO_APP_CLIENT_ID =
 	process.env.EGO_APP_RDPC_CLIENT_ID || 'donor-submission-aggregator';
 export const EGO_APP_CLIENT_SECRET = process.env.EGO_APP_RDPC_CLIENT_SECRET || 'top-secret';
 
-export const EGO_PUBLIC_KEY = process.env.EGO_PUBLIC_KEY as string;
+export const EGO_PUBLIC_KEY =
+	process.env.EGO_PUBLIC_KEY ||
+	`-----BEGIN PUBLIC KEY-----\r\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0lOqMuPLCVusc6szklNXQL1FHhSkEgR7An+8BllBqTsRHM4bRYosseGFCbYPn8r8FsWuMDtxp0CwTyMQR2PCbJ740DdpbE1KC6jAfZxqcBete7gP0tooJtbvnA6X4vNpG4ukhtUoN9DzNOO0eqMU0Rgyy5HjERdYEWkwTNB30i9I+nHFOSj4MGLBSxNlnuo3keeomCRgtimCx+L/K3HNo0QHTG1J7RzLVAchfQT0lu3pUJ8kB+UM6/6NG+fVyysJyRZ9gadsr4gvHHckw8oUBp2tHvqBEkEdY+rt1Mf5jppt7JUV7HAPLB/qR5jhALY2FX/8MN+lPLmb/nLQQichVQIDAQAB\r\n-----END PUBLIC KEY-----`;
```

Comment:
This is an environment specific value and should not default to any specific value. I assume this is the Ego default public key from its source, which is not a problematic value to put here, but it would be best if the software never assumes what public key to use. It should require one is provided in the env. If you want to put this as a default value in teh env schema that could be useful.

Reply:
This comment should also apply to the other env variables... but a) those are already in and we don't need to perpetuate a problematic pattern, and b) those are easy to identify as being wrong due to the `localhost` or obviously incorrect default values - while this default value is hard to identify as a default since it is complex and so looks like a legit key.

Reply:
that makes sense. I copied from https://github.com/icgc-argo/platform-api/blob/develop/src/config.ts#L75